shared/desktop/app/index.js

```diff
@@ -24,6 +25,8 @@ function start() {
       if (mainWindow) {
         mainWindow.show(true)
       }
+      // Wasn't showing on Windows without this
+      showMainWindow()
```

Comment:
OK I made it windows-only for now, but I suspect the previous `mainWindow.show(true)` didn't work on other platforms either, likely because `mainWindow` isn't set yet.